python/chariots/core/metadata.py

```diff
@@ -0,0 +1,40 @@
+import operator
+import copy
+from typing import List
+
+class Metadata:
```

Comment:
add comment to explain that this is a remain and that it should not be used as such